src/components/Table/GridTable.stories.tsx

```diff
@@ -3,6 +3,7 @@ import { observable } from "mobx";
 import { Fragment, useEffect, useMemo, useRef, useState } from "react";
 import {
   actionColumn,
+  beamFlexibleStyle,
```

Comment:
Might be unused?

Reply:
Heh, it was while testing. Forgot to autoformat 😅